CHANGELOG.md

```diff
@@ -25,6 +25,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.21.5] - 2023-05-08
+
+### Fixed
+- Updated container build script to pull in the correct kubectl version from the
+  metal-provision repository instead of the csm-rpms repository.
```

Comment:
It might be worth retroactively adding this to our 3.22.0 version in the integration and main branches. I think it's fine to revise and clarify content of an older changelog version entry as long as we are not changing the code in that released version.